android/build.gradle

```diff
@@ -26,15 +26,13 @@ allprojects {
 }
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
-
-
 android {
-  compileSdkVersion 23
-  buildToolsVersion "23.0.3"
+  compileSdkVersion 27
+  buildToolsVersion "27.1.1"
```

Comment:
`27.1.1` is not valid check [this page](https://stackoverflow.com/questions/49921807/failed-to-find-build-tools-revision-27-1-1-when-i-try-to-download-says-it-has) .. must be 
`buildToolsVersion '27.0.3'`